sdk/formrecognizer/Azure.AI.FormRecognizer/src/USReceiptType.cs

```diff
@@ -1,35 +1,40 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-using System;
-
 namespace Azure.AI.FormRecognizer.Models
 {
     /// <summary>
+    /// The receipt type of recognized <see cref="USReceipt"/> instances.
     /// </summary>
     public enum USReceiptType
```

Comment:
I wasn't really sure about what to put here, so pay extra attention when reviewing this file.